README.md

```
@@ -1,81 +1,175 @@
+## Table of Contents
+
+- [About](#about)
+- [Status](#status)
+  - [Master Branch](#master-branch)
+  - [Stable Branch](#stable-branch)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Features](#features)
+  - [Modules](#modules)
+  - [Scripts](#scripts)
+  - [Documentation](#documentation)
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)